Triangle120.java

```diff
@@ -0,0 +1,29 @@
+import java.util.List;
+//time complexity : O(n)
+//space complexity : O(1)
```

Comment:
We are using additional space for output array and we are not returning it right? So, space complexity is not constant here